binary-serialise-cbor/binary-serialise-cbor.cabal

```diff
@@ -51,7 +51,7 @@ library
   exposed-modules: Data.Binary.Serialise.CBOR.Read
 
   build-depends:
-    base       >= 4.11 && < 4.19,
+    base       >= 4.11 && < 4.21,
```

Comment:
Why `< 4.21` and not `< 4.20`?  Is there evidence that it works with GHC-9.10?
If one follows a strict regime here, one should stick to the versions tested by CI.

Reply:
`ghc-9.10` will use `base-4.21`.

`Base` skips odd numbers just like `ghc` skips odd numbers.

Reply:
Fixed.